examples/smtp_selfsigned.rs

```diff
@@ -39,6 +39,6 @@ fn main() {
     // Send the email
     match mailer.send(&email) {
         Ok(_) => println!("Email sent successfully!"),
-        Err(e) => panic!("Could not send email: {:?}", e),
+        Err(e) => panic!("Could not send email: {e:?}"),
```

Comment:
I initially did it because the CI was broken on my PR (clippy errors), so I fixed them. But I agree it could be better in another PR, should I move it?

Reply:
If CI is failing I guess makes sense to keep it.

Reply:
:heavy_check_mark: